### ISSUE TITLE
Found unsupported images

### ISSUE THREAD

**Issue by VoetenK** [Chibald/maestrogateway#39]
i'm running Home Assistant Supervised on debian 11 Bullseye installed on a raspberry pi 4b. 
I have installed Docker version 20.10.12, build e91ed57 and docker-compose version 1.29.2. 
I installed MaestroGateway as explained.
Home Assistant supervisor log reports the following warning:
12/21/28 14:27:21 WARNING (MainThread) [supervisor.resolution.evaluations.base] Found unsupported images: {'chibald/maestrogateway'} (more info: https://www.home-assistant.io /more-info/unsupported/software)
What's going wrong?

**Comment by VoetenK** [comment]
Update:
I used install_daemon in a ternminal for my maestrogateway installation.
There is an issue between MCZ's MQTT and Home Assistant's MQTT Broker in my setup. My suspicion is because the maestro.service is started before Home assistant is finished with its MQTT Brocker, they cannot communicate with each other.
I have therefore modified my crontab file.
sudo crontab -e
Add to this following.
@reboot (sleep 30 && service maestro start)
Now everything works perfectly.
I used docker container first. It also started up but also didn't communicate. I suspect the same problem is going on here. I can't solve that myself.
